test/controllers/mod_warning_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ModWarningControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
```

Comment:
I don't know what this means but it looks very meta. :-)

Reply:
It means, that I haven't had time to write any tests...